roscala/src/main/scala/coop/rchain/rosette/VirtualMachine.scala

```diff
@@ -239,27 +246,38 @@ object VirtualMachine {
     }
 
     if (mState.doRtnFlag) {
-      //if (mState.ctxt.ret(mState.ctxt.rslt)) {
-      //  mState = mState.set(_ >> 'vmErrorFlag)(true)
-      //} else if (mState.doRtnFlag) {
-      //  mState = mState.set(_ >> 'doNextThreadFlag)(true)
-      //}
+      // may set doNextThreadFlag
+      mState = doRtn(mState).set(_ >> 'doRtnFlag)(false)
     }
 
     if (mState.vmErrorFlag) {
-      handleVirtualMachineError(mState)
+      //handleVirtualMachineError(mState)
```

Comment:
Does this disable error handling, or are errors handled elsewhere?

Reply:
`handleVirtualMachineError(mState)` doesn't do anything currently because `OprnVmError` in `Ctxt.vmError(...)` is still a placeholder. I will add a comment to revisit this code when we have `OprnVmError` working.